tests/features/java/gc.feature

```diff
@@ -53,3 +53,19 @@ Feature: Openshift OpenJDK GC tests
     Then s2i build log should contain Using MAVEN_OPTS -XX:+UseG1GC
     And container log should contain -XX:+UseG1GC
     And container log should not contain -XX:+UseParallelGC
+
+  Scenario: Check GC_METASPACE_SIZE GC configuration
+    Given s2i build https://github.com/jboss-openshift/openshift-quickstarts from undertow-servlet
+       | variable                 | value  |
+       | GC_METASPACE_SIZE    | 120    |
+    Then s2i build log should contain Using MAVEN_OPTS -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:MetaspaceSize=120m
```

Comment:
For this test, we only care about the `-XX:MetaspaceSize=120m` value, right? I'd just add that. Same for the assertion below (i.e. container log). Finally, a `And container log should not contain integer expression expected` would be nice.

Reply:
Thank-you. Fixed.